sympy/sets/__init__.py

```diff
@@ -3,3 +3,7 @@
 from .fancysets import ImageSet, Range, ComplexRegion
 from .contains import Contains
 from .conditionset import ConditionSet
+
+from ..core.singleton import S
```

Comment:
I think it should stay as a relative import.

This error only occurs with Sphinx. Maybe there is a bug in Sphinx.